generator/transit_generator.cpp

```diff
@@ -1,16 +1,146 @@
 #include "generator/transit_generator.hpp"
 
+#include "generator/osm_id.hpp"
+
+#include "routing_common/transit_serdes.hpp"
+#include "routing_common/transit_types.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/file_name_utils.hpp"
+#include "coding/file_writer.hpp"
+
+#include "platform/platform.hpp"
+
+#include "base/checked_cast.hpp"
 #include "base/logging.hpp"
 #include "base/macros.hpp"
 
+#include "3party/jansson/myjansson.hpp"
+
+using namespace platform;
+using namespace std;
+
+namespace
+{
+using namespace routing;
+using namespace routing::transit;
+
+/// \returns file name without extension by a file path if a file name which have zero, one of several extensions.
```

Comment:
zero, one of several extensions?

I didn't get the comment, please, rephrase.

Reply:
Done:
```
/// \returns file name without extension(s) by a file path. That means it gets a file name by a full path and
/// and removes from the file name the first dot and everything after the dot, if the file name contains dot(s).
/// For example,
/// GetFileName("Russia_Nizhny Novgorod Oblast.transit.json") returns "Russia_Nizhny Novgorod Oblast"
/// GetFileName("Russia_Nizhny Novgorod Oblast.mwm") returns "Russia_Nizhny Novgorod Oblast"
```